aspnetcore/fundamentals/startup.md

```diff
@@ -170,16 +66,8 @@ Middleware execution order is set by the order of `IStartupFilter` registrations
   * Position the service registration before the library is added to the service container.
   * To invoke afterward, position the service registration after the library is added.
 
-## Add configuration at startup from an external assembly
-
-An <xref:Microsoft.AspNetCore.Hosting.IHostingStartup> implementation allows adding enhancements to an app at startup from an external assembly outside of the app's `Startup` class. For more information, see <xref:fundamentals/configuration/platform-specific-configuration>.
```

Comment:
I know we cannot access the app's `Startup`, but can we still mention IHostingStartup an link to https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/platform-specific-configuration?view=aspnetcore-7.0 in the .NET 7 docs? That should still work with WebApplicationBuilder